docs/explain.md

```diff
@@ -0,0 +1,46 @@
+
+Here's the rendered example:
+
+## [uvular muscle](http://purl.obolibrary.org/obo/UBERON_0010235) SubClassOf [muscle organ](http://purl.obolibrary.org/obo/UBERON_0001630) and ([part of](http://purl.obolibrary.org/obo/BFO_0000050) some [jaw region](http://purl.obolibrary.org/obo/UBERON_0011595)) ##
+
+  - [uvular muscle](http://purl.obolibrary.org/obo/UBERON_0010235) SubClassOf [palatal muscle](http://purl.obolibrary.org/obo/UBERON_0003682)
+    - [palatal muscle](http://purl.obolibrary.org/obo/UBERON_0003682) EquivalentTo [skeletal muscle organ](http://purl.obolibrary.org/obo/UBERON_0014892) and ([part of](http://purl.obolibrary.org/obo/BFO_0000050) some [soft palate](http://purl.obolibrary.org/obo/UBERON_0001733))
+      - [soft palate](http://purl.obolibrary.org/obo/UBERON_0001733) SubClassOf [part of](http://purl.obolibrary.org/obo/BFO_0000050) some [secondary palate](http://purl.obolibrary.org/obo/UBERON_0001716)
+        - [secondary palate](http://purl.obolibrary.org/obo/UBERON_0001716) SubClassOf [part of](http://purl.obolibrary.org/obo/BFO_0000050) some [upper jaw region](http://purl.obolibrary.org/obo/UBERON_0001709)
+          - [upper jaw region](http://purl.obolibrary.org/obo/UBERON_0001709) SubClassOf [part of](http://purl.obolibrary.org/obo/BFO_0000050) some [jaw region](http://purl.obolibrary.org/obo/UBERON_0011595)
+      -  Transitive: [part of](http://purl.obolibrary.org/obo/BFO_0000050)
+      - [skeletal muscle organ](http://purl.obolibrary.org/obo/UBERON_0014892) EquivalentTo [muscle organ](http://purl.obolibrary.org/obo/UBERON_0001630) and ([develops_from](http://purl.obolibrary.org/obo/RO_0002202) some [myotome](http://purl.obolibrary.org/obo/UBERON_0003082)) and ([surrounded_by](http://purl.obolibrary.org/obo/RO_0002219) some [epimysium](http://purl.obolibrary.org/obo/UBERON_0011899)) and ([composed primarily of](http://purl.obolibrary.org/obo/RO_0002473) some [skeletal muscle tissue](http://purl.obolibrary.org/obo/UBERON_0001134)) and ([synapsed by](http://purl.obolibrary.org/obo/uberon/core#synapsed_by) some [motor neuron](http://purl.obolibrary.org/obo/CL_0000100))
+
+In addition to outputting the explanation axioms in Markdown, an ontology containing just 
+those axioms is provided to any subsequent chained commands:
+
+    robot explain -i uvula.ofn -r ELK --axiom "'uvular muscle' SubClassOf 'muscle organ' and 'part of' some 'jaw region'" convert -o uvular_muscle.ofn
+
+You can still output a Markdown file, if desired, before the chained command.
```

Comment:
An example would be good (it wouldn't have to be part of the test cases if you surround it with ticks instead of the indentation).